lib/actions/preload.js

```diff
@@ -239,10 +239,11 @@ export const preload = {
 	description: 'preload an app on a disk image (or Edison zip archive)',
 	help: `\
 Preload a balena application release (app images/containers), and optionally
-a balenaOS splash screen, in a previously downloaded balenaOS image file (or
-Edison zip archive) in the local disk. The balenaOS image file can then be
-flashed to a device's SD card.  When the device boots, it will not need to
-download the application, as it was preloaded.
+a balenaOS splash screen, in a previously downloaded '.img' balenaOS image file
+in the local disk (a zip file is only accepted for the Intel Edison device type).
+After preloading, the balenaOS image file can be flashed to a device's SD card.
+When the device boots, it will not need to download the application, as it was
+preloaded.
```

Comment:
Separate commit, partially addressing #1810, trying to make it clearer that `balena preload` requires a `.img` file instead of `.zip` for most device types. Re automatically unzipping for the user, the tricky bit is deciding the device type before unzipping, given that the device type may be determined by inspecting the image -- which requires the image to be in the correct format for that device type to begin with. There is another issue for that though: https://github.com/balena-io-modules/balena-preload/issues/156